HelpSource/Classes/Stethoscope.schelp

```diff
@@ -172,6 +172,23 @@ METHOD:: style
         One of the above Integers.
 
 
+SUBSECTION:: Size and position
```

Comment:
This probably doesn't need it's own subsection, but can be included in the previous subsection `SUBSECTION:: Display`, above or below the documentation for the `size` method.

Reply:
OK, first I did what you wrote, and then I thought maybe it needed a new subsection. However, it is a bit too complicated. I changed it as you described.
https://github.com/supercollider/supercollider/pull/6281/commits/39802c61144acfa132c8c72b41da4d118490cddb